ext/RMagick/rminfo.c

```diff
@@ -1641,6 +1641,7 @@ Info_group_eq(VALUE self, VALUE value)
     rb_check_frozen(self);
     Data_Get_Struct(self, Info, info);
     info->group = R_long_to_C_long(value);
+    return self;
```

Comment:
@Watson1978 in Ruby shouldn't setter methods return the object passed in?

```c
return value;
```

Reply:
This is correct: the standard `attr_writer` behaviour is that the passed in object is returned.

Reply:
right. the implementation is wrong.

Reply:
Okay, thanks for the clarification. It shouldn't matter hopefully, since these are deprecated anyway.

Reply:
These are not the only ones where that is an issue. Just create #597 to change them all.